thirdparties/glog/glog.patch

```diff
@@ -1006,3 +1006,45 @@ index 9554718..e02bd02 100644
  };
  
  static bool kFailureSignalHandlerInstalled = false;
+diff --git a/src/vlog_is_on.cc b/src/vlog_is_on.cc
+index e8fdbae..eb3a39e 100644
+--- a/src/vlog_is_on.cc
++++ b/src/vlog_is_on.cc
+@@ -123,7 +123,9 @@ struct VModuleInfo {
+ };
+ 
+ // This protects the following global variables.
+-static Mutex vmodule_lock;
```

Comment:
there's exists more than one global `Mutex`, you should check whether the same problem will happen on other global `Mutex`.

Reply:
done